### ISSUE TITLE
Facebook Login Error 3840 [iOS]

### ISSUE THREAD

**Issue by francocorreasosa** [ParsePlatform/parse-server#49]
Using Parse and ParseUI with parse-server (production) gives me a login error. I have migrated all the data and I set the serverURL on app delegate. 

It was working on Parse Hosted.

![liginerroroarseserver](https://cloud.githubusercontent.com/assets/4152942/12687502/15c1c092-c6af-11e5-902f-e0d5c9ecc045.png)

The HTTP App Transport is correctly configured to enable non-HTTPS requests.

Logs:
```
2016-01-29 17:33:03.604 HispaLearn[74822:3733505] -canOpenURL: failed for URL: "fbauth2:/" - error: "(null)"
2016-01-29 17:33:03.611 HispaLearn[74822:3733505] -canOpenURL: failed for URL: "fbauth2:/" - error: "(null)"
2016-01-29 17:33:47.800 HispaLearn[74822:3733505] -canOpenURL: failed for URL: "fbauth2:/" - error: "(null)"
2016-01-29 17:33:47.813 HispaLearn[74822:3733505] -canOpenURL: failed for URL: "fbauth2:/" - error: "(null)"
2016-01-29 17:33:57.914 HispaLearn[74822:3733505] -canOpenURL: failed for URL: "fbauth2:/" - error: "(null)"
2016-01-29 17:33:57.927 HispaLearn[74822:3733505] -canOpenURL: failed for URL: "fbauth2:/" - error: "(null)"
```

**Comment by francocorreasosa** [comment]
Updated error description:
```
3840
Error Domain=NSCocoaErrorDomain Code=3840 "JSON text did not start with array or object and option to allow fragments not set." UserInfo={NSDebugDescription=JSON text did not start with array or object and option to allow fragments not set.}
```

**Comment by gfosco** [comment]
Because the iOS app is failing on 'canOpenURL' for a 'fbauth2' url, it seems like this is not parse-server related...

**Comment by francocorreasosa** [comment]
But with parse hosted works. Read the other log, something about JSON format.

**Comment by gfosco** [comment]
If you can capture any of the data, that would be useful... hard to see where it could be failing.

**Comment by francocorreasosa** [comment]
I think you fixed it with the Facebook app ID pull request. I will confirm that tomorrow in order to get this issue closed.

**Comment by francocorreasosa** [comment]
How can I get the data returned by the server with the iOS SDK? (Swift 2)

**Comment by francocorreasosa** [comment]
It's working now, you fixed it with #86 
Feel free to close this issue, thanks!
